solutions/devicesimulation-nohub/armtemplate/template.json

```diff
@@ -197,7 +197,8 @@
             "type": "string",
             "metadata": {
                 "description": "Unique Id of the deployment."
-            }
+            },
+            "defaultValue": "Undefined"
```

Comment:
Now if we don't provide the deployment id we will use undefined and then if we receive logs from these deployments will get ignored. I think the id should be always provided and the fix should be in the calling code. Otherwise, this will give rise to errors that will be hard to track.

Reply:
For now we are sending "undefined" as deployment id with the logs. Logs will not be ignored. Generating and saving deployment id is tracked as a separate change in suite.